analogout.py

```diff
@@ -94,10 +102,10 @@ def load_xml(self, node):
                     self.physicalChannels = child.text
 
                 elif child.tag == "minimum":
-                    self.minValue = float(child.text)
+                    self.minValue = str_to_float(child.text)
```

Comment:
str_to_float should be float. I made an error when merging master into my branch and i will fix this